classifiers/image/CLIPZeroShotClassifier/config.yml

```
@@ -0,0 +1,7 @@
+!CLIPZeroShotClassifier
+with:
+  {}
```

This might fix the docker container failure.
```suggestion
  labels: ['cat', 'dog']
```